examples/DataApi-Water-And-Energy-Usage/README.md

```diff
@@ -1,57 +1,83 @@
-
 # Metagraph - Data API - Water and Energy Usage
 
-  
-
 This is one example of the Data API, where we can send transactions to update the state of Water and Energy usage
-  
 
 ## Template
 
 This project contains dependencies of the Tessellation on version `develop`
 
- 
 At this example you can take a look at the files:
 
 `modules/l0/src/main/scala/com/my/currency/l0/Main.scala`
+
 `modules/l1/src/main/scala/com/my/currency/l1/Main.scala`
+
 `modules/data_l1/src/main/scala/com/my/currency/data_l1/Main.scala`
-`modules/shared_data/src/main/scala/com/my/currency/shared_data/Main.scala`
 
+`modules/shared_data/src/main/scala/com/my/currency/shared_data/Main.scala`
 
 ### shared_data
-->   `validateData` and `validateUpdate`
-* These functions will check the provided data to verify its validity. If the amount that we want to update  of water and energy usage is valid (at this example > 0), we will proceed to the state update, otherwise, we will return an error
+
+-> `validateData`
+
+- This function will check the provided data to verify its validity. This validation happens on the L0 layer. In this case, we consider the `oldState` and the `proofs`. For this example, we check if the provided address is the same that the address that signed the message. We also check the `oldState` to avoid doubling when updating
```

Comment:
```suggestion
- This function will check the provided data to verify its validity. This validation happens on the L0 layer. In this case, we consider the `oldState` and the `proofs`. For this example, we check that the provided address is the same as the address that signed the message. We also check the most recent update timestamp in order to prevent old data or duplicated data from being accepted.
```